frontend/src/app/core/services/files-downloader.service.ts

```diff
@@ -0,0 +1,49 @@
+import { HttpEventType, HttpResponse } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+import { HttpInternalService } from './http-internal.service';
+import { NotificationService } from './notification.service';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class FilesDownloaderService {
+    private readonly staticFilesRoutePrefix = '/api/staticfiles';
+
+    // eslint-disable-next-line no-empty-function
+    constructor(private httpClient: HttpInternalService, private notificationService: NotificationService) {}
+
+    public downloadSquirrelInstaller() {
+        this.httpClient.getFullBlobRequest(`${this.staticFilesRoutePrefix}/downloadSquirrelInstaller`)
+            .subscribe({
```

Comment:
please unsubscribe

Reply:
Thanks, I will resolve this. Regarding your comment above, as stated in the ticket for this task, for now, we are putting it in manually, and later, it will be included in the build process.